hasjob/views/listing.py

```diff
@@ -953,6 +925,19 @@ def newjob():
         if g.user:
             # form.poster_name.data = g.user.fullname  # Deprecated 2013-11-20
             form.poster_email.data = g.user.email
+
+    # Job Reposting
+    if request.method == 'GET' and request.args.get('template'):
+        archived_post = JobPost.get(request.args.get('template'))
```

Comment:
This can be `request.args['template']` since it's presence in `request.args` is already established.